cypress/README.md

```diff
@@ -27,54 +27,40 @@ Actual tests can be found in `cypress/integration/ui/`.
-* TODO `cy.treeSelect('All VMs & Templates', 'HyperV', 'SCVMM')` - switch tree items
-* `cy.toolbar('Configuration', 'Edit this VM')` - trigger toolbar buttons
-* TODO `cy.tab('Report Info')` - switch miq tabs
+##### explorer
 
-TODO allow using {id:...} instead of string label for menu items, gtl items, tree nodes, accordions, toolbar items
+* `cy.accordion(title)` - open an accordion panel. `title`: String for the accordion title for the accordion panel to open.
+* `cy.accordionItem(name)` - click on a record in the accordion panel. `name`: String for the record to click in the accordion panel.
 
-##### inspection
+##### gtl
 
-* `cy.menuItems()` - parse the main menu and yield an object tree
-* `cy.toolbarItem(...)` - yields an object describing a toolbar button state, or null
+* `cy.gtl_error()` - check that error message is present.
+* `cy.gtl_no_record()` - check that `No Record` message is present.
+* `cy.gtlGetTable()` - returns GTL table.
+* `cy.gtlGetRows(columns)` - return GTL table row data in an array. `columns`: Array of 0-based indexes of the columns to read (e.g. [1, 2, 3] will return all row data from columns 1, 2, and 3).
+* `cy.gtlClickRow(columns)` - click on a row in a GTL table. `columns`: Array of `{ title: String, number: Integer }`. `title` is the string you want to find in the table to click on, `number` is the column that string is found in. (e.g. `[{title: 'Default', number: 1}, {title: 'Compute', number: 2}]` will click on a row in the GTL table with `Default` in column 1 and `Compute` in column 2. Using just `[{title: 'Default', number: 1}]` will click on the first row found in the GTL table with `Default` in column 1).
 
-#### assertions
+##### login
 
-* `cy.expect_explorer_title('Active Services')` - check the title on an explorer screen
-* `cy.expect_show_list_title('Cloud Providers')` - check the title on a show\_list screen
-* `cy.expect_search_box()` - check if searchbox is present on screen
-* `cy.expect_no_search_box()` - check if no searchbox is present on the screen
-* `cy.expect_rates_table(headers, rows)` - check the values in a chargeback rate table. `headers`: Array of strings representing the headers of the table. `rows`: Array of type `[String, [...String], [...String], [...String], [...String], String]` where each index of the array represents a column in the table. The arrays within the `rows` array can be any length and represent the values in each given column, e.g. an array of `[0.0, 100.0]` in the index for the `Range Start` column would verify that the column contains two range starts with values `0.0` and `100.0`.
-* TODO `cy.expect_layout('miq-layout-center_div_with_listnav')` - check current layout
+`cy.login(user = admin, password = smartvm)` - log in to ManageIQ with the provided username and password. `user`: String for the user account to log in to, default is `admin`. `password`: String for the user account password to log in with, default is `smartvm`.
```

Comment:
Even though it's only a single item, prefix this with `* ` for consistency with the other sections - otherwise it renders funny:

![manageiq-ui-classic_cypress_README_md_at_3f0c4c4607bb5d5c26e1e6900c780a07d3953d93_·_ManageIQ_manageiq-ui-classic](https://github.com/ManageIQ/manageiq-ui-classic/assets/52120/3a5528b3-7d69-4529-9ae4-c76f12139f49)

Reply:
Should be fixed now